### PR TITLE
Is MongoDB Python driver v4.6.3 an official release?

### PR DESCRIPTION
**Pull request by justinclift**
# Is the MongoDB Python Driver 4.6.3 release a real one?

Created because this Git Repo has "Issues" disabled, and neither the MongoDB Security Team nor MongDB Support have responded when asked if the claimed recent "4.6.3" release is actually legit.

The *documented* release process for this Python driver is here:

  https://github.com/mongodb/mongo-python-driver/blob/master/RELEASE.md

However, the MongoDB "4.6.3" release which GitHub Dependabot is telling people to upgrade to IS NOT listed where that says to check:

  * MongoDB Release Announcements page:

      https://www.mongodb.com/community/forums/tags/c/announcements/driver-releases/110/python

    Nope.  Latest release listed is 4.6.2.

  * GitHub Releases page for the repo:

      https://github.com/mongodb/mongo-python-driver/releases

    Nope.  Latest release listed is 4.6.2.

In short, this looks like *some* part of the MongoDB packaging process may have been compromised.

However, the MongoDB staff are unfortunately incapable of telling people whether it's legit or not.

Seems like a pretty basic question eh?

If anyone from the MongoDB team actually feels like responding to let their users know, our particular issue documenting the attempts to contact them is here: https://github.com/getredash/redash/pull/6863

### CODE REVIEW

**Comment by ShaneHarvey** [comment]
Yes 4.6.3 is an official release which fixes a bug in the C BSON decoding layer. Apologies for the confusion around the release announcements. We will publish the release announcement in our regular channels soon.

**Comment by justinclift** [comment]
Thanks @ShaneHarvey, that helps. :smile:

Out of curiosity, what went wrong with the Release Announcements this time around?

**Comment by justinclift** [comment]
Also, is there a way to point people at the broken support page to see about getting it fixed?

That breakage is documented here, including screenshot: https://github.com/getredash/redash/pull/6863#issuecomment-2046202375

**Comment by behackett** [comment]
Hi @justinclift, we use Jira for ticket tracking, documented in these places:
https://github.com/mongodb/mongo-python-driver?tab=readme-ov-file#bugs--feature-requests
https://pymongo.readthedocs.io/en/stable/#getting-help

There is also a link to report potential security issues here:
https://github.com/mongodb/mongo-python-driver?tab=readme-ov-file#security-vulnerabilities

Can you tell me how you were directed to https://www.mongodb.com/company/thank-you/contact? Was it after using one of the contact forms here:
https://www.mongodb.com/company/contact

**Comment by justinclift** [comment]
@behackett Thanks for following up:

> we use Jira for ticket tracking ...

Your Jira installation requires people to create an account.  I'm personally not willing to do that for a simple once-off question. :smile:


> There is also a link to report potential security issues here:

That's quite literally the process I followed several days ago:

image

The input I used for that form is documented [here](https://github.com/getredash/redash/pull/6863#issuecomment-2043769609).

There has been exactly zero response from that.  Not a thing.


> Can you tell me how you were directed to ...

Sure.  By using the web form on that [exact contact page](https://www.mongodb.com/company/contact):

image
